src/patterns/EmbeddedContentAttribute.tsx

```diff
@@ -62,6 +63,8 @@ export const EmbeddedContentAttribute = ({
     );
   }
 
+  src = transformValue(src);
```

Comment:
This looks to be missing from the `EmbeddedContentAttributeProps` interface, also is it required/should we check if it's undefined before calling it?

Reply:
Fixed in 8800262. 👍